### ISSUE TITLE
Aave: Add Reserves index page

### ISSUE THREAD

**Issue by kenkunz** [tradingstrategy-ai/frontend#211]
**Epic:** tradingstrategy-ai/oracle#116
**Depends on:** #215 

# Goal

Add a new `frontend` page that displays all lending reserves that we collect data on. Initially, this will only be Aave.

## Details

- page route: `/trading-view/reserves`
- backend API: tbd (see tradingstrategy-ai/backend#66)
- follow the UI patterns used on `/trading-view/exchanges` and `trading-view/trading-pairs`
- target URL for row: `/trading-view/[chain]/reserve/[reserve]` (see #215)
- remember to add this new page to `sitemap.xml`

**Comment by plamut** [comment]
> Add a new `frontend` page that displays all lending pools that we collect data on. Initially, this will only be Aave.

@kenkunz FYI, the term "pool" is ambiguous, since on the backend, the term "pool" is often a synonym a "liquidity pool", i.e. a "reserve" in AAVE v3 terms. To avoid confusion, we decided to ban this word, and the new terminology is the following:
- We have a **lending protocol** (the name not final yet) that represents a decentralized place where assets are lent / borrowed.
- In a lending protocol , we can find multiple **reserves**. A reserve is a "pile of a particular asset" that can be borrowed, or lent to the reserve to increase the latter's liquidity.

---

TL; DR - The API will provide a list of _reserves_ for a particular protocol (only AAVE for the time being). We avoid the term ~~pool~~ on both the backend and the frontend to avoid confusion.
